atomic-data-mons/url-lanthanidesV-VII.txt

```diff
@@ -0,0 +1,4 @@
+https://doi.org/10.5281/zenodo.10635803
```

Comment:
Please add direct URLs for the zip files that are needed as we do for other data sources.

Reply:
I'm not sure what the links are. Does it work to just add the files names to the end of the zenodo link?

Reply:
https://zenodo.org/records/10635803/files/outggf_Ln_V--VII.zip
https://zenodo.org/records/10635803/files/outglv_Ln_V--VII.zip
The readme should probably just be committed directly to the repo since it's so small.

Reply:
The info from the readme is in comments at the top of the file